src/lib/services/settings-service.ts

```diff
@@ -9,7 +9,7 @@ export const fetchSettings = async (request = fetch): Promise<Settings> => {
 
   return {
     auth: {
-      enabled: settings.Auth.Enabled,
+      enabled: settings?.Auth?.Enabled,
```

Comment:
This will be null instead of false. We probably want a true/false value here (even though it'll probably work in js for most if statements if someone does if auth.enabled == false we'll have a bad time) I would recommend adding

`settings?.Auth?.Enabled ?? false`

Reply:
Or you can use a `!!` to coerce it into a boolean.